.github/workflows/dependency-check-paths-ignore.yml

```diff
@@ -19,6 +19,13 @@ jobs:
   Go:
     runs-on: ubuntu-latest
     steps:
-      - run: 'echo "No job required" '
-
+      - name: Collect Metrics
```

Comment:
Should we ignore this for any of the `-ignore.yml` workflows?

Reply:
I think we could probably skip them as they're just filler to make GitHub required checks happy.